app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number", 0),
+        "column": error.get("column_number", 0),
+        "message": error.get("text", ""),
+        "name": error.get("code", ""),
```

Comment:
It would be applicable to leave the default output of `None` in case the key is not present in the dict. But kudos to populating default values that could apply to the content!